parser_library/test/expressions/arithmetic_expression_test.cpp

```diff
@@ -250,9 +250,9 @@ TEST(arithmetic_expressions, multiple_operand_with_spaces)
 )";
     analyzer a(input);
     a.analyze();
-
     a.collect_diags();
-    ASSERT_EQ(a.diags().size(), (size_t)3);
+
+    EXPECT_TRUE(matches_message_codes(a.diags(), { "S0002", "S0002", "S0002", "CE012", "CE012" }));
```

Comment:
Just curious - the two `CE012` errors point to the following lines?

```
&A2 SETA FIND( 'A','B')
&A3 SETA FIND('A','B' )
```

And `&A1 SETA FIND( 'A','B' )` doesn't throw `CE012` error as the statement inside of the brackets couldn't be parsed at all due to prefixed and suffixed spaces?

Reply:
afaik antlr takes only `FIND` as the argument, the rest is then marked by the syntax error, except in the last case, where it actually starts parsing the parenthesis when the attempt fails, I think it drops the whole argument.

Reply:
Thx!